source/linear_algebra/matrix.hpp

```diff
@@ -27,9 +27,7 @@ class Matrix{
         inline Matrix operator/(const int & that) const {return *this / float(that);};
         inline Matrix operator-() {return *this * -1;};
         Matrix operator*(const Matrix & that) const;
-        Vector operator*(const Vector & that) const; 
-        Matrix operator^(const int & that) const;
```

Comment:
I couldn't figure out how to do powers of matrices although I want to. (positive integer powers)

Reply:
This isnt needed for now, if we ever have a need for it we can add it in